spyder/app/cli_options.py

```diff
@@ -37,5 +37,8 @@ def get_options():
                            "not related with Python profiling)")
     parser.add_option('--window-title', type=str, default=None,
                       help="String to show in the main window title")
+    parser.add_option('--project', default=None, type=str,
```

Comment:
Do we have a way of opening a file? that would be useful as well

spyder somefile.py